dev-docs/modules/userId.md

```diff
@@ -74,44 +74,34 @@ of sub-objects. The table below has the options that are common across ID system
 
 ## User ID Sub-Modules
 
-### Criteo RTUS
+### Criteo ID
 
-Criteo Real Time User Sync (RTUS) is designed for use as an alternative for platforms that cannot drop their cookies due to Safari 3rd party restriction.
+Criteo ID module is designed for use as an alternative for platforms that cannot drop their cookies due to Safari 3rd party restriction.
```

Comment:
That description is invalid or obsolete, here's a suggestion:

[Criteo](https://criteo.com) is the leading advertising platform for the Open Internet. The Criteo ID module enables browser-level, device-level and person-level campaigns from the Criteo platform to achieve optimal take rate on all web browsers.